### DISCUSSION TITLE
Installer Fails with RDP Errors 14 and 15 on Fresh VM Setup

### DISCUSSION THREAD

**Discussion by Fort-P** [winapps-org/winapps#354] [Support] [answered]
### Description
I am encountering a recurring issue while trying to set up the installer on a fresh VM. The same behavior occurs whether I use libvirt or docker, so it seems unrelated to the virtualization method.

Case 1: The installer exits with code 14, after briefly flashing a black login loading window (too brief for me to capture a SS of it), Error:
```
ERROR: REMOTE DESKTOP PROTOCOL FAILURE.
FreeRDP failed to establish a connection with Windows.
```
Case 2: Without any changes to the VM, it connects, flashes the window multiple times, and then errors out with code 15:
```
ERROR: APPLICATION QUERY FAILURE.
Failed to query Windows for installed applications.
```

No winapps.log file is generated. However, the FreeRDP log is dependent on the error thrown:
[Case 1](https://gist.github.com/Fort-P/46b71b345544e19ec6a7db1d5bb5f352)
[Case 2](https://gist.github.com/Fort-P/8bb2e298b49d0318ec0ba5bfdc0e567c)

### Configuration
The config file is based on the example configuration but has comments removed for brevity:
```
RDP_USER="Docker"

RDP_PASS=""

RDP_DOMAIN=""

RDP_IP="127.0.0.1"

WAFLAVOR="docker"

RDP_SCALE="100"

RDP_FLAGS="/cert:tofu /sound /microphone /network:lan"

MULTIMON="false"

DEBUG="true"

AUTOPAUSE="off"

AUTOPAUSE_TIME="300"

FREERDP_COMMAND=""
```

### Steps to Reproduce
Perform a fresh VM install (tested with both libvirt and docker).
Attempt to launch the installer.
Observe the errors and behavior described above.

### Expected Behavior
The installer should proceed without exiting with errors.

### Actual Behavior
Error codes 14 and 15 occur as described, preventing proper functionality.

Any guidance on how to troubleshoot this or resolve the issue would be greatly appreciated!

**Comment by oskardotglobal** [comment, marked as answer]
Does it make a difference if you remove the `/network:lan` flag from the config?

**Reply by Fort-P** [reply]
That does fix the issue. What's that flag for if it's just causing issues?
Thank you so much!

**Reply by oskardotglobal** [reply]
It improves performance.

**Comment by Yanis002** [comment]
Hi, I got the same issue and I searched a fix, the previous post marked as the answer worked for me, I'm thinking this should probably be written somewhere (like in a troubleshooting FAQ for instance) if not done already

**Reply by oskardotglobal** [reply]
I've added it to the config file comments.
